Missing instructions for building documentation locally

### What is the expected enhancement?

We should add instructions for building documentation locally for contributors who want to contribute on (non jupyter notebook) documentation such as doc string and .rst files.

I could do the local build writing `make html` when on the docs folder of the qiskit-machine-learning (cloned) repo as the contributing guideline https://github.com/Qiskit/qiskit-machine-learning/blob/main/CONTRIBUTING.md indicated and it worked just fine.

However I had already installed `pandoc` so I tried removing it and redoing the local build. This time I got the following error:

```Notebook error:
PandocMissing in tutorials/01_neural_networks.ipynb:
Pandoc wasn't found.
Please check that pandoc is installed:
https://pandoc.org/installing.html
make: *** [html] Error 2
```



So `pandoc` is needed here.

--- a/CONTRIBUTING.md
+++ b/CONTRIBUTING.md
@@ -51,11 +51,7 @@ please ensure that:
    
    The documentation will be built/tested using Sphinx and should be free
    from errors and warnings.
-   Make sure you have `pandoc` installed:
-   ```
-   MacOS: brew install pandoc
-   Ubuntu: sudo apt-get -y install pandoc
-   ```
+   You will need to [install pandoc](https://pandoc.org/installing.html) first.
    
    Then you can run
    ```shell script

--- a/CONTRIBUTING.md
+++ b/CONTRIBUTING.md
@@ -10,7 +10,12 @@ https://qiskit.org/documentation/contributing_to_qiskit.html
 ## Contributing to Qiskit Machine Learning
 
 In addition to the general guidelines above there are specific details for
-contributing to Qiskit Machine Learning, these are documented below.
+contributing to Qiskit Machine Learning.
+
+You should first install the python development libraries by running
+`pip install -r requirements-dev.txt` from the root of the
+Machine Learning repository clone and then
+follow the  guidelines below.
 
 ### Project Code Style.
 
@@ -46,12 +51,17 @@ please ensure that:
    
    The documentation will be built/tested using Sphinx and should be free
    from errors and warnings.
+   Make sure you have `pandoc` installed:
+   ```
+   MacOS: brew install pandoc
+   Ubuntu: sudo apt-get -y install pandoc
+   ```
    
-   You can run
+   Then you can run
    ```shell script
     make html
    ```
-   in the 'docs' folder. You might also like to check the html output
+   from the root of the Machine Learning repository clone. You might also like to check the html output
    to see the changes formatted output is as expected. You will find an index.html
    file in docs\_build\html and you can navigate from there.
    

